plugins/modules/zos_backup_restore.py

```diff
@@ -124,7 +124,7 @@
     required: True
   recover:
     description:
-      - Specifies if potentially recoverable errors should be ignored.
+      - When I(operation=backup) specifies if potentially recoverable errors should be ignored.
```

Comment:
My comment is to expand the docs clarity, this comment really applies to all doc,  imagine you bring no experience and are reading this for the first time is how I approach doc. 
I see you referenced I(operation=backup) in the option which is great, my suggestion is below. 

Hence:
```
  recover:
    description:
      - When I(recover=true) and I(operation=backup) then potentially recoverable errors will be ignored.
    type: bool
    default: False
```